src/lib.rs

```diff
@@ -69,7 +69,13 @@ impl Display for Number {
     let fixed_value = self.value + d128!(0);
     let output = match self.unit {
       Unit::NoUnit => format!("{}", fixed_value),
-      unit => format!("{} {:?}", fixed_value, unit),
+      unit => {
+        if self.value > d128!(1) {
+          format!("{} {}s", self.value, unit.to_string())
```

Comment:
Maybe instead of ` >` we should use `!=`

Reply:
This would solve the issue with 0.

Reply:
Either way you would still end up with "kilometers per hours"